### PR TITLE
HOLD: Remove django mail logger

### PR DESCRIPTION
**Pull request by mbeacom**
##### Description
The goal of this PR is to remove the Django `mail_logger` and replace it with direct CloudWatch logging for persistent logs that we can setup alarms on.

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
logging

##### Testing
Locally

### CODE REVIEW

File: app/app/settings.py [hunk 1]
```diff
@@ -189,37 +190,63 @@
...
+        aws_access_key_id=AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=AWS_SECRET_ACCESS_KEY,
+        region_name=AWS_DEFAULT_REGION
+    )
+
     LOGGING = {
         'version': 1,
         'disable_existing_loggers': False,
+        'root': {
+            'level': 'ERROR',
```
Comment (SaptakS) [inline app/app/settings.py]: So we are not logging the warnings? 😅

Comment (mbeacom) [reply app/app/settings.py]: This is the root/default logging behavior and aligns with the default logger provided by Django as far as I know. 🤔

Comment (mbeacom) [reply app/app/settings.py]: Watchtower is set to log whatever the level is that's passed to: `AWS_LOG_LEVEL` in this case: `DEBUG` - So, much more verbose than what we're even logging currently. @SaptakS

Comment (SaptakS) [reply app/app/settings.py]: Okay. Makes sense.

**Pull request closed without merging**